packages/merge-styles/src/mergeStyleSets.test.ts

```diff
@@ -149,6 +149,21 @@ describe('mergeStyleSets', () => {
     expect(_stylesheet.getRules()).toEqual('.root-0{background:red;}' + '.root-1{background:green;}');
   });
 
+  it('can merge correctly when class names are provided by application', () => {
```

Comment:
can you elaborate why this is testing for what you are fixing? what's the test failure looks like without your fix?

Reply:
talked with @xugao offline, w/o my change, it will fail on
style = mergeStyleSets({ test: ['test-0'] }),  expect(styles.test).toBe('test-0');  
Because it will generate an extra empty style, so the result will be Received: "test-0 ns-test-0"